options.ggo

```diff
@@ -10,4 +10,8 @@ option "port" p "rtl_tcp port"
 option "host" h "rtl_tcp host address"
   string typestr="address" default="localhost" optional
 option "listen" l "Listening port for client connections"
-  int typestr="port" default="7878" optional
+  int typestr="port" default="7878" optional option "delayed" d "Delayed 
+connection to the server"
```

Comment:
This line wrapping (and the next one) causes some issues in formatting for the --help and comments in the cmdline handler.